src/cleanup_rules/swift/rules.toml

```diff
@@ -769,6 +769,39 @@ not_contains = [
     )""",
 ]
 
+[[rules.filters]]
+not_enclosing_node = "(value_argument_label) @va"
+
+[[rules]]
+name = "replace_argument_with_value"
+query = """(
+        (value_argument
+            name: (value_argument_label)
+            value: [
+                (simple_identifier) @revar @valere
+                (navigation_expression
+                    target: (self_expression)
+                    suffix: (navigation_suffix
+                        suffix: (simple_identifier) @revar
+                    )
+                ) @valere
+                (ternary_expression
```

Comment:
do we still need this ternary expression pattern here?

Reply:
Yep, for the cases where arguments are passed on the basis of ternary condition evaluation.